examples/texture-arrays/uniform.frag

```diff
@@ -0,0 +1,15 @@
+#version 450
+
+layout(location = 0) in vec2 v_TexCoord;
+// layout(location = 1) flat in int v_Index;  // dynamically non-uniform
+layout(location = 0) out vec4 o_Color;
+
+layout(set = 0, binding = 0) uniform texture2D u_Textures[2];
+layout(set = 0, binding = 1) uniform sampler u_Sampler;
+layout(set = 1, binding = 0) uniform Uniforms {
+    int u_Index;  // dynamically uniform
```

Comment:
wait, is this not "unsized" as well?

Reply:
Sized vs unsized is about if `uniform texture2D u_Textures[2];` has a compile time array size or not.